tests/tests.cmake

```diff
@@ -15,6 +15,7 @@ add_subdirectory(streamtests)
 
 add_executable( unittests
     unittests/unittests.cc
+    unittests/unittests.cpp
```

Comment:
Having `unitests.cpp` and `unittests.cc` is confusing. Please use a different name for the new file, e.g. `app_unittests.cc` or `image_unittests.cc` or similar.

Reply:
fixed in https://github.com/KhronosGroup/KTX-Software/pull/493/commits/812bd2b237df53b865f3ddedd1409a040d164afa